test/config/basic/basicConfig.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const { stat } = require('fs');
+const { resolve } = require('path');
+const { run, extractSummary } = require('../../utils/test-utils');
+
+describe('config flag test : Basic config file', () => {
+    it('Should not throw any module not found', done => {
+        const { stdout } = run(__dirname, ['-c', resolve(__dirname, 'webpack.config.js')]);
+        const summary = extractSummary(stdout);
+        const outputDir = 'basic/bin';
+        const outDirectoryFromCompiler = summary['Output Directory'];
+        const outDirToMatch = outDirectoryFromCompiler
```

Comment:
This is a test done in Windows environment. This will fail in Unix environment. I'd suggest to test it in other way

Reply:
done